setup_undercloud.yml

```diff
@@ -67,30 +48,46 @@
         - name: set undercloud to PXE boot (Supermicro)
           shell: ipmitool -I lanplus -H mgmt-{{ undercloud_hostname }} -U quads -P {{ chassis_password }} chassis bootdev pxe options=persistent
           when: lab_name == "scale" and undercloud_hostname.split('.')[0].split('-')[3] in scale.machine_types.supermicro
+          delegate_to: "{{ hammer_host }}"
+          vars:
+            ansible_user: root
+            ansible_python_interpreter: "{{ hammer_python_interpreter }}"
```

Comment:
As the task to get the python interpreter is removed in the latest changes. hammer_python_interpreter var is undefined